manifests/profile/neutron/agent.pp

```diff
@@ -7,4 +7,28 @@
     default:    { include ::openstack::common::ml2::ovs }
   }
 
+  ### Router service installation
+  class { '::neutron::agents::l3':
+    debug                    => $::openstack::config::debug,
+    external_network_bridge  => 'brex',
+    enabled                  => true,
```

Comment:
Hi @mmailand,

could you make these option/variables to be retrieved from YAML file?
let user can change the bridge name, or switching between enable and disable

thanks